pints/_diagnostics.py

```diff
@@ -0,0 +1,60 @@
+#
+# Functions to calculate various MCMC diagnostics
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+import numpy as np
+
+
+def autocorrelation(lx):
```

Comment:
Why do all the variable names start with "l"? Don't think we should be using these conventions in Pints